WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -125,19 +125,33 @@ struct CollapsibleProductRowCard: View {
         }
         .cornerRadius(Layout.frameCornerRadius)
     }
+
+
+}
+
+private extension CollapsibleProductRowCard {
+    func trackAddDiscountTapped() {
```

Comment:
Suggestion here. Even if it's done like that in other cases, it's more optimal if the view doesn't know what's the logic of the view model (tracking). In this case the view model function would be `onAddDiscountTapped`, hiding the tracking purpose.